packages/zoe/src/contracts/priceAggregatorChainlink.js

```diff
@@ -434,7 +434,7 @@ const start = async zcf => {
     const rrId = reportingRoundId;
 
     let canSupersede = true;
-    if (_roundId !== 1n) {
+    if (_roundId > 1n) {
```

Comment:
Why change this?

Is it a huge hassle to add a unit test for whatever is going on here?

Reply:
IOU unit tests when we actually start using this contract.  As it stands, you're seeing my attempts to keep the types working while trying to debug what is going on.

Reply:
ok... I see this refs #3595 but doesn't claim to close it.